### PR TITLE
BUG: Legend box order

### PR DESCRIPTION
**Pull request by aflaxman**
This hacky PR adjusts the zorder of the bbox in a legend so that it appears behind the markers, fixing an issue with filled markers (reported here http://stackoverflow.com/questions/31351947/mpld3-generated-html-missing-xticklabels-and-color-in-the-legend).
Before:
image

After:
image

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - source activate testenv
+  - conda install --yes $DEPS
+  - python setup.py install
+
+before_install:
+  # setup virtual x
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  # then install python version to test
+  - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "2" ]; then wget http://repo.continuum.io/miniconda/Miniconda-2.2.2-Linux-x86_64.sh -O miniconda.sh; else wget http://repo.continuum.io/miniconda/Miniconda3-2.2.2-Linux-x86_64.sh -O miniconda.sh; fi
```
Comment (damianavila) [inline .travis.yml]: You can get the latest miniconda from here: https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh, then your change to get an old conda is lower... althought the conda update conda is something reasonable to do...

Comment (aflaxman) [inline .travis.yml]: @damianavila : Thanks, that sounds like a good idea.  I've tried to break this out into a separate PR and make the change you propose in #41.  Can you let me know there if this is what you meant?

Comment (damianavila) [inline .travis.yml]: Answered in #41
